examples/wgpu-example-compute-runner/src/main.rs

```diff
@@ -0,0 +1,121 @@
+
+async fn create_device_queue() -> (wgpu::Device, wgpu::Queue) {
+    let backend = wgpu::BackendBit::all();
+    let instance = wgpu::Instance::new(backend);
+        
+    let adapter = instance
+        .request_adapter(&wgpu::RequestAdapterOptions {
+            power_preference: wgpu::PowerPreference::default(),
+            compatible_surface: None,
+        })
+        .await
+        .expect("Failed to find an appropriate adapter");
+        
+    adapter
+        .request_device(
+            &wgpu::DeviceDescriptor {
+                features: wgpu::Features::empty(),
+                limits: wgpu::Limits::default(),
+                shader_validation: true,
+            },
+            None,
+        )
+        .await
+        .unwrap()
+}
+
+macro_rules! include_bytes_align_as {
+    ($align_ty:ty, $file:expr) => {{
+        #[repr(C)]
+        pub struct AlignedAs<Align, Bytes: ?Sized> {
+            pub _align: [Align; 0],
+            pub bytes: Bytes,
+        }
+
+        static ALIGNED: &AlignedAs::<$align_ty, [u8]> = &AlignedAs {
+            _align: [],
+            bytes: *include_bytes!($file),
+        };
+
+        &ALIGNED.bytes
+    }};
+}
+
+fn disassemble_spirv(binary: impl AsRef<[u32]>) -> String {
+    use rspirv::binary::Disassemble;
+    let mut loader = rspirv::dr::Loader::new();
+    rspirv::binary::parse_words(binary, &mut loader).unwrap();
+    loader.module()
+        .disassemble()
+}
+
+fn main() {
+    let spirv = include_bytes_align_as!(u32, env!("wgpu_example_compute_shader.spv"));
+    println!("{}\n", disassemble_spirv(bytemuck::cast_slice(spirv)));
```

Comment:
prooobably shouldn't include this, as all the other examples don't? but shrug, doesn't really matter either way.